frontend/pulse3d/src/pages/_app.js

```diff
@@ -117,6 +117,12 @@ function Pulse({ Component, pageProps }) {
     return () => clearInterval(swInterval);
   }, [router.pathname]);
 
+  useEffect(() => {
+    //clear default file when user leaves the re-analyze page
+    if (router.query.id !== "Re-analyze Existing Upload") {
+      setDefaultReanalysisFile(null);
+    }
+  }, [router.query]);
```

Comment:
Can this go in the uploads form component itself? Or does it need to be in this top level comp?